mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -1401,10 +1407,47 @@ void expungeShouldCallAllPreDeletionHooksOnEachMessageDeletionCall() throws Exce
                     .containsOnly(composeId1.getMessageId(), composeId2.getMessageId());
             }
 
+            @Test
+            void expungeShouldCallAllPreDeletionHooksOnlyOnMessagesMarkedAsDeleted() throws Exception {
+                ComposedMessageId composeId1 = inboxManager.appendMessage(AppendCommand.builder()
```

Comment:
Can you add a message without DELETED?

Reply:
Yes, and I did it in an other test :trollface:

Reply:
I asked that because it's what the test name means, IMO.

Reply:
Yep looks like a intellij warning that got solved and I removed the all append... sorry and good catch..